benchmarks/apps/sphng/sphng.py

```diff
@@ -0,0 +1,223 @@
+import os.path as path
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+
+class SphngBase(SpackTest):
+
+    descr = 'Base class for Sphng'
+    valid_systems = ['*']
+    valid_prog_environs = ['*']
+    spack_spec = 'sphng@v1.0.0'
```

Comment:
```suggestion
    spack_spec = 'sphng@v1.0.0%intel'
```
Although we generally try to avoid it, I think here it would make sense to hard-code the intel compiler to the spec since it's the only supported compiler and the package recipe will fail with any other compiler.

Reply:
Technically I suppose the `oneapi` compiler would also work, but I would expect to always have an `intel` compiler available if `oneapi` is installed.

Reply:
OK I will add `intel`. the user will have to overwrite the recipe if `intel` is not available.

Reply:
Thanks, I think that's better than user always having to overwrite the recipe if `intel` is not the default!